curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-060.md

```diff
@@ -0,0 +1,286 @@
+---
+id: 6201a1cc668a34888f5b2f52
+title: Step 60
+challengeType: 0
+dashedName: step-60
+---
+
+# --description--
+
+Within your `tr` elements, select the elements with the `class` set to `data`. Give them a background image of `linear-gradient(to bottom, #dfdfe2 1.845rem, white 1.845rem)`.
```

Comment:
maybe it's me not being native speaker, but I first thought it was asking me to select an element inside `tr`, not `tr` elements with a certain class

Reply:
```suggestion
Select the `tr` elements with the `class` set to `data`. Give them a background image of `linear-gradient(to bottom, #dfdfe2 1.845rem, white 1.845rem)`.
```